crypto/ml_dsa/ml_dsa_sign.c

```diff
@@ -258,15 +371,7 @@ static int ml_dsa_verify_internal(const ML_DSA_KEY *pub, int msg_is_mu,
     if (!ossl_ml_dsa_sig_decode(&sig, sig_enc, sig_enc_len, pub->params)
             || !matrix_expand_A(md_ctx, pub->shake128_md, pub->rho, &a_ntt))
         goto err;
-    if (msg_is_mu) {
-        if (msg_enc_len != mu_len)
-            goto err;
-        mu_ptr = (uint8_t *)msg_enc;
-    } else {
-        if (!shake_xof_2(md_ctx, pub->shake256_md, pub->tr, sizeof(pub->tr),
-                         msg_enc, msg_enc_len, mu_ptr, mu_len))
-            goto err;
-    }
+
     /* Compute verifiers challenge c_ntt = NTT(SampleInBall(c_tilde) */
```

Comment:
```suggestion
    /* Compute verifiers challenge c_ntt = NTT(SampleInBall(c_tilde)) */
```

Reply:
yes, this one needs closing :)

Reply:
FWIW, though I am not forcing the issue, I don't think that leaving the open paren is a particularly effective way to outline the logic.  Instead just just list what's going to be hashed in the comment, or refer to the appropriate FIPS 203 section, and then just let the code clearly match the comment.

Reply:
The general M' logic is already explained in the function description, if you want me to remove the comments thru the function I can do that, I have no attachments to code.

Reply:
Do what you think is best, but if the comments are redundant, feel free to drop them, but keep them if they're useful...